.github/workflows/release.yml

```diff
@@ -25,3 +25,39 @@ jobs:
         run: |
           python3 -m build --sdist --wheel .
           twine upload dist/*
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pipenv
+          pipenv install --dev --system --skip-lock
+          pip install https://github.com/Toloka/stubmaker/archive/refs/heads/main.zip
+      - name: Checkout docs repository
+        uses: actions/checkout@v3
+        with:
+          repository: 'Toloka/docs'
+          path: 'docs-repo'
+      - name: Generate markdowns
+        run: |
+          python ./misc/make_markdowns.py --module-root crowdkit --src-root crowdkit --output-dir docs/reference \
+            --described-objects misc/described_objects_crowdkit.py \
+            --github-source-url https://github.com/Toloka/crowd-kit/blob/${{ github.event.release.tag_name }}/crowdkit
```

Comment:
I think it is safer to use `--github-source-url "https://github.com/Toloka/crowd-kit/blob/${{ github.event.release.tag_name }}/crowdkit"` to avoid issues with whitespaces.